docs/index.md

```diff
@@ -201,13 +201,13 @@ To do this, open your `/ios/{projectName}/AppDelegate.mm` file (or `AppDelegate.
 
 At the top of the file, import the Firebase SDK right after `'#import "AppDelegate.h"'`:
 
-```
+```objectivec
```

Comment:
I think github uses `obj-c` but I've tried this before, the system we use to generate the docs doesn't work with that and requires `objectivec`, there's an example in messaging/usage.md which I always have to look at from my previous work to remember which to use, and `objectivec` is it